edge/stack/stack.go

```diff
@@ -196,6 +198,19 @@ func (manager *StackManager) Start() error {
 				ctx := context.TODO()
 
 				if stack.Action == actionDeploy || stack.Action == actionUpdate {
+					for stackImage, cacheImage := range stack.ImageMapping {
```

Comment:
Maybe something better than `StackImagesURLs` can be used, but at least renaming the variables would make the purpose of this method clearer I think
```suggestion
	for stackImageName, imageCacheURL := range stack. StackImagesURLs {
```